### PR TITLE
new version of Sfp 403 nl

### PR DESCRIPTION
**Pull request by ToonHub**
## Description

This is the second version of the protocol dating from 18/11/2022.

## Task list

Steps by contributor:

-   [x] Add description to this pull request (under "## Description")
-   [ ] Open the dropdown (triangle) near 'create pull request' and choose 'Create draft pull request'
-   [x] Check for potential problems by running `protocolhelper::check_frontmatter()` and address them
-   [x] Check for potential problems by running `protocolhelper::check_structure()` and address them
-   [x] Add further commits if needed and push them to GitHub
-   [x] Update the protocol-specific `NEWS.Rmd`
-   [x] Mark the pull request as 'ready for review'

Review steps for the author(s):

-   [x] Add reviewers, at least one subject-matter specialist and one administrator
-   [x] Wait for review comments and address them
-   [ ] Iterate until reviewer approvals (merging the pull request will be done by an administrator)
-   [ ] Verify that the checks done by continuous integration succeeded. These will check if `protocolhelper::check_frontmatter()` and `protocolhelper::check_structure()` succeeded without errors.

To be done by an administrator after review: see [guidelines for admins](https://github.com/inbo/protocolsource/blob/main/RELEASES.md).

### CODE REVIEW

**Comment by ToonHub** [comment]
I still have to deal with

- reference to protocol sfp-118 which still have to be added to protocolsource
- reference to protocols that are in development

**Comment by ToonHub** [comment]
Below you find the rendered protocol. Unzip and select index.html to view.
[sfp_403_nl_vegopname_lsvi_3260.zip](https://github.com/inbo/protocolsource/files/11562140/sfp_403_nl_vegopname_lsvi_3260.zip)

**Comment by anleyssen** [comment]
I screened “index.html” and the pdf-version. Some comments:
- link of ‘deze tabel’ in paragraph ‘metadata’ doesn’t seem to work
- I suppose ‘Afhankelijkheden’ should be completed with the other sfp’s? 
- strange symbols in paragraph 7.2.2 ‘Rolmeter’:  SmartMeasure[^06_benodigdheden-2]
- some references need a hard return at the end; so e.g. Environment Agency. (2003), Denys L. (2011), Osté, … begin on a new line
- is it possible to add clickable links for the other protocols (e.g. sfp-113); probably work for the future, as these are not yet available on the github-page  

Other parts seem to be ok.

**Comment by hansvancalster** [comment]
> * link of ‘deze tabel’ in paragraph ‘metadata’ doesn’t seem to work

This link will work when it is published on the website. It just does not work when you preview a protocol in isolation from the other stuff on the website.

**Comment by ToonHub** [comment]
An, thanks for the review.

I corrected following issues

 >  - strange symbols in paragraph 7.2.2 ‘Rolmeter’: SmartMeasure[^06_benodigdheden-2]
> - some references need a hard return at the end; so e.g. Environment Agency. (2003), Denys L. (2011), Osté, … begin on a new line

I will add the protocol sfp-113 to test linking to other protocols.

**Comment by ToonHub** [comment]
@anleyssen I added a link to [protocol sfp-113-nl](https://inbo.github.io/protocols/2023.04/index.html). You are welcome to approve the pull request :-)

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/07_werkwijze.Rmd [hunk 1]
```diff
@@ -230,7 +230,7 @@ Indien de schijf tot op de bodem zichtbaar is, wordt 'bodemzicht' aangevinkt.
 De meting van de secchi-diepte dient in het midden van de waterloop uitgevoerd te worden ter hoogte van de plaats waar ook het dwarsprofiel wordt opgemeten.
 Bij te grote diepte, kan de secchi-diepte bepaald worden vanop een brug.
 Wanneer door stroming de secchi-schijf sterk afdrijft en de schijf niet loodrecht onder het wateroppervlak komt te hangen, kan de secchi-schijf aan de onderzijde verzwaard worden.
-Zie ook deelprotocol (sfp-113).
+Zie ook protocol [sfp-113](https://inbo.github.io/protocols/2023.04/index.html).
```
Comment (hansvancalster) [inline source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/07_werkwijze.Rmd:233]: Deze link relatief maken en ondubbelzinnig aangeven om welk protocol het gaat:

Zie ook protocol [sfp-113-nl 2023.04](../2023.04/index.html).

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/09_veiligheid.Rmd [hunk 2]
```diff
@@ -18,11 +18,10 @@ Alle aandachtspunten in verband met veilig werken.
...
+    In deze omstandigheden kan een alternatieve opnametechniek toegepast worden (zie \@ref(vegopn)).
     Gebruik een reddingstouw of reddingsvest wanneer de situatie dit vereist.
 
 Tijdens het veldwerk gelden volgende aanvullende veiligheidsregels:
 
--   algemene veiligheidsregels rond het werken in en nabij water (sfp-112);
--   bioveiligheidsmaatregelen voor het voorkomen van de verspreiding van invasieve exoten (sfp-015).
-
+-   algemene veiligheidsregels rond het werken in en nabij water (protocol in ontwikkeling sfp-112);
+-   bioveiligheidsmaatregelen voor het voorkomen van de verspreiding van invasieve exoten (protocol in ontwikkeling sfp-015).
```
Comment (hansvancalster) [inline source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/09_veiligheid.Rmd:27]: Kan er al naar deze protocols gelinkt worden? Eventueel oude website?

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/05_competenties.Rmd [hunk 3]
```diff
@@ -6,4 +6,11 @@ Omschrijving van de basiskennis (bv. kennis van bepaalde soorten) en vaardighede
 Eventueel kan er verwezen worden naar bepaalde opleidingen.
 -->
 ```
-Voldoende kennis van de veldkenmerken van macrofyten die in waterlopen worden aangetroffen en vertrouwd zijn met technieken om ze te kunnen identificeren.
+Voldoende kennis van de veldkenmerken van macrofyten die in waterlopen worden aangetroffen, vertrouwd zijn met technieken om ze te kunnen identificeren en vertrouwd zijn met de deelprotocol sfp-015, sfp-112, sfp-113 en sfp-118.
```
Comment (hansvancalster) [inline source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/05_competenties.Rmd:9]: Hier ook weblinks toevoegen indien mogelijk.

**Review by hansvancalster — APPROVED** [review]
Behalve de ontbrekende weblinks lijkt alles ok. Best altijd eens lokaal render_protocol() uitvoeren om te checken of alles er OK uit ziet.

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/07_werkwijze.Rmd (hunk 1 above)
Comment (ToonHub) [reply source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/07_werkwijze.Rmd:233]: OK

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/09_veiligheid.Rmd (hunk 2 above)
Comment (ToonHub) [reply source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/09_veiligheid.Rmd:27]: Die protocols moeten nog geschreven worden. Ik laat het hier staan zodat het niet vergeten wordt.

File: source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/05_competenties.Rmd (hunk 3 above)
Comment (ToonHub) [reply source/sfp/4_vegetation/sfp_403_nl_vegopname_lsvi_3260/05_competenties.Rmd:9]: De te ontwikkelen protocols heb ik hier verwijderd.

**Pull request merged**